targets/mediatek-filogic

```diff
@@ -50,6 +53,17 @@ device('netgear-wax220', 'netgear_wax220', {
 	factory_ext = '.img',
 })
 
+-- OpenWRT
+device('openwrt-one', 'openwrt_one',{
+	factory = '-factory',
+	factory_ext = '.ubi',
+	sysupgrade_ext = '.itb',
+	extra_images = {
+		{'-nor-factory', '-nor-factory', '.bin'},
```

Comment:
@blocktrron, @ambassador86 Is the NOR intended to be used by us, or just to be a recovery means?

Reply:
NOR is for recovery, so I'd also suggest dropping it.

Background: NOT image is for reflashing NAND, which Gluon currently does not support. One could detect the boot-source in gluon and only offer reflashing from there, but I'd reserve this for later.